apps/examples/wifi_manager/wm_test/wm_test_best_rssi.md

```diff
@@ -0,0 +1,13 @@
+Setup guide:
+a)Build TP1x source code with rtl8721csm
+b)H/W Requirement:
+	1)Multiple Access Points, some of them have same credential (same SSID & password)
+	2)Device under test (DUT) – TP1x board - Simulating Samsung DA device
+	
+Test guide: 
+wm_test connect_by_rssi [SSID_NAME] [SSID_authentication] [SSID_Password] [Repeat Count]
+e.g. wm_test connect_by_rssi TP-Link_Hrishikesh wpa2_aes 20212021 3
+
+This Test will scan various access points , for a given SSID (having multiple access points),
+it will check TP1x board is connected to access point having best RSSI value.
```

Comment:
Please check this file in `view file`. This looks strange.

Reply:
Done